easybuild/easyconfigs/n/networkx/networkx-2.5.1_decorator.patch

```diff
@@ -0,0 +1,21 @@
+diff -Naur networkx-2.5.1.orig/networkx.egg-info/requires.txt networkx-2.5.1/networkx.egg-info/requires.txt
```

Comment:
@robert-mijakovic could you add a comment about the purpose of the patch please?

Reply:
I don't think networkx 2.5.1 is relevant any longer. networkx 2.6.1 is released and it doesn't require a patch. I will update the pull request.